docs/examples.rst

```diff
@@ -190,6 +190,36 @@ L1 Metadata Specification::
     pass directory path must be defined
 
 
+Inherit Tests
```

Comment:
From words 'Inherit Tests' I'd assume using same set of tests across multiple plans.

How about "Share common attributes"?

Reply:
Agreed, maybe even "Share common test attributes"? Though I am not sure if that isn't too long and verbose.

Reply:
"Share common attributes" sounds good!